src/App.tsx

```diff
@@ -141,10 +149,20 @@ export const App: React.FC = () => {
 
       {/* Notification is shown in case of any error */}
       {/* Add the 'hidden' class to hide the message smoothly */}
-      <div className="notification is-danger is-light has-text-weight-normal">
-        <button type="button" className="delete" />
+      <div
+        className={error
+          ? 'notification is-danger is-light has-text-weight-normal'
+          : 'notification is-danger is-light has-text-weight-normal hidden'}
+      >
+        <button
+          type="button"
+          className="delete"
+          onClick={() => setError(false)}
```

Comment:
now we will reset an error message after click:

`onClick={() => setErrorMessage('')}`